libs/community/langchain_community/chat_models/__init__.py

```diff
@@ -241,6 +249,9 @@
     "ChatMLX": "langchain_community.chat_models.mlx",
     "ChatMaritalk": "langchain_community.chat_models.maritalk",
     "ChatMlflow": "langchain_community.chat_models.mlflow",
+    "ChatOCIModelDeploymentEndpoint": "langchain_community.chat_models.oci_data_science_model_deployment_endpoint",  # noqa: E501
```

Comment:
I think it is better to use a shorter filename instead of `noqa: E501`. Maybe just `oci_data_science` or `oci_model_deployment`?

Reply:
Sure. This file name is following the name used in llms. But yes, we can change it.

Reply:
Looks like we still need `noqa: E501` even with the shorter name `oci_data_science`.  Probably we need to shorter the class name. Do we want to remove endpoint in the class name? 
 ![Screenshot 2024-07-24 at 10 00 23](https://github.com/user-attachments/assets/64b0454a-4716-4986-9bdd-2817d51ce8f5)